src/Aux_.cpp

```diff
@@ -135,34 +144,30 @@ void Aux::step() {
 		sl2 -= fb2 * rr1;		
 	}
 
-	outputs[SEND1L_OUTPUT].value = sl1;
-	outputs[SEND1R_OUTPUT].value = sr1;
+	outputs[SEND1L_OUTPUT].setVoltage(sl1);
+	outputs[SEND1R_OUTPUT].setVoltage(sr1);
 
-	outputs[SEND2L_OUTPUT].value = sl2;
-	outputs[SEND2R_OUTPUT].value = sr2;
+	outputs[SEND2L_OUTPUT].setVoltage(sl2);
+	outputs[SEND2R_OUTPUT].setVoltage(sr2);
 
 	if (!bypass) {
-		outl += params[RETURN1_PARAM].value * rl1;
-		outr += params[RETURN1_PARAM].value * rr1;		
-		outl += params[RETURN2_PARAM].value * rl2;
-		outr += params[RETURN2_PARAM].value * rr2;		
+		outl += params[RETURN1_PARAM].getValue() * rl1;
+		outr += params[RETURN1_PARAM].getValue() * rr1;		
+		outl += params[RETURN2_PARAM].getValue() * rl2;
+		outr += params[RETURN2_PARAM].getValue() * rr2;		
 	}
 
-	outputs[OUTL_OUTPUT].value = outl;
-	outputs[OUTR_OUTPUT].value = outr;
+	outputs[OUTL_OUTPUT].setVoltage(outl);
+	outputs[OUTR_OUTPUT].setVoltage(outr);
 }
 
 struct AuxWidget : ModuleWidget { 
-	AuxWidget(Aux *module) : ModuleWidget(module) {
+	AuxWidget(Aux *module) {
+		setModule(module);
 
 		box.size = Vec(15*4, 380);
 
-		{
-			SVGPanel *panel = new SVGPanel();
-			panel->setBackground(SVG::load(assetPlugin(plugin, "res/Aux_.svg")));
-			panel->box.size = box.size;
-			addChild(panel);	
-		}
+    setPanel(APP->window->loadSvg(asset::plugin(pluginInstance, "res/Aux_.svg")));
```

Comment:
You can do this in other modules as well. I realized after a while that these were deprecation warnings but weren't harming anything.